src/main/java/qna/domain/Question.java

```diff
@@ -106,13 +112,9 @@ public DeleteHistories delete(User loginUser) throws CannotDeleteException {
 
     private DeleteHistories generateHistories(List<DeleteHistory> histories) {
         return DeleteHistories.of()
-                .addHistory(DeleteHistory.builder()
-                        .contentType(ContentType.QUESTION)
-                        .contentId(getId())
-                        .deletedBy(getWriter())
-                        .createDate(LocalDateTime.now())
-                        .build())
+                .addHistory(DeleteHistory.of(ContentType.QUESTION, getId(), getWriter(), LocalDateTime.now()))
```

Comment:
인수를 좀 더 줄여볼 수 있을꺼 같아요! 😄 
```suggestion
                .addHistory(DeleteHistory.ofQuestion(getId(), getWriter()));
```

Reply:
아 인자를 더 줄일 수 있겠네요.
적용해보겠습니다. 감사합니다. 👍